tests/integration/expected_test_results/page_crawler/normal-user-access-nested-facet-page.html

```diff
@@ -10,7 +10,7 @@
 	<meta property="fb:app_id" content="219331381518041">
     <meta property="og:type" content="food">
     <meta property="og:title" content="">
-    <meta property="og:url" content="//world.openfoodfacts.localhost/category/en:hazelnut-spreads/brand/nutella">
+    <meta property="og:url" content="//world.openfoodfacts.localhost/facets/categories/en:hazelnut-spreads/brands/nutella/1">
```

Comment:
not a big deal, but we probably need the same rules you added to remove the en: and the 1. (the en: issue preexisted before your change)

Reply:
Honestly for a canonical ur, I think it make sense to be really explicit.